3-platform/warehouse/infra/src/main/java/com/_6core/platform/warehouseinfra/adapter/driven/persistence/entity/InventoryEntity.java

```diff
@@ -0,0 +1,33 @@
+package com._6core.platform.warehouseinfra.adapter.driven.persistence.entity;
+
+import java.util.Optional;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Column;
+import org.springframework.data.relational.core.mapping.Table;
+
+@Data
```

Comment:
Dont use @Data for Entity. 
```suggestion
@Setter
@Getter
```